babel/core.py

```diff
@@ -60,6 +60,9 @@ def get_global(key):
             fileobj.close()
     return _global_data.get(key, {})
 
+# FIXME: Very ugly hack in order to break cyclic import dependency
+localedata.get_global = get_global
```

Comment:
with that I'm saying we probably want to file a bug about this as this is a code architectural issue and not in scope for this PR.